src/typechecker/base.lisp

```diff
@@ -60,6 +61,8 @@ This requires a valid PPRINT-VARIABLE-CONTEXT")
      (with-pprint-variable-scope ()
        ,@body)))
 
+(defvar *pprint-aliases* nil)
```

Comment:
should have docs if exported

Reply:
resolved in [9b3beda](https://github.com/coalton-lang/coalton/pull/1294/commits/9b3beda0bbc78b5dd029a0f229c4c08f8203bd7f)